examples/karpenter/main.tf

```diff
@@ -14,6 +14,10 @@ terraform {
       source  = "hashicorp/helm"
       version = ">= 2.4.1"
     }
+    kubectl = {
+      source  = "gavinbunney/kubectl"
+      version = ">= 1.13.1"
+    }
```

Comment:
Are we okay to use a 3P provider? Or should we consider using [tfk8s](https://github.com/jrhouston/tfk8s) which is Terraform [recommended](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/manifest)?

Reply:
I think we agreed to proceed with `gavinbunney/kubectl` as this works better than the Hashiciorp Kubernetes manifest. We introduced this in our TEAMS feature. This can handle any kubernetes object including the CRDs.

Hashicorp Kubernetes manifest resources has few limitations which is causing an issues for our plan. It has limitation ons CRDs as well. check this out https://www.hashicorp.com/blog/beta-support-for-crds-in-the-terraform-provider-for-kubernetes. 

yes, for now we will be using kubectl_manifest until we have better version from Hashicorp